cerbero/build/recipe.py

```diff
@@ -123,6 +123,7 @@ class Recipe(FilesProvider):
     deps = None
     platform_deps = None
     force = False
+    library_name_override = None
```

Comment:
If you add a new variable make sure you document it in the class as the rest of them